src/videojs-http-streaming.js

```diff
@@ -597,6 +597,7 @@ class VhsHandler extends Component {
     this.options_.customTagParsers = this.options_.customTagParsers || [];
     this.options_.customTagMappers = this.options_.customTagMappers || [];
     this.options_.cacheEncryptionKeys = this.options_.cacheEncryptionKeys || false;
+    this.options_.llhls = this.options_.llhls === false ? false : true;
```

Comment:
I know we use this style above, but I think it's clearer as:

```suggestion
    this.options_.llhls = Boolean(this.options_.llhls);
```

Reply:
Coming back to this, Kevin has restored the previous state:
```js
this.options_.llhls === false ? false : true
```
This is because simply doing `Boolean(this.options_.llhls)` was not defaulting to `true`, which is what was wanted.